### PR TITLE
Created a database and a table

### PR DESCRIPTION
**Pull request by kennankole**
Created `vet_clinic` database and `animal` table
Populated the database table
Performed `SQL` queries on the database table. 

### `SQL`  query results;
Agumon-or-Pikachu
all-neutered
born-between-2016-and-2019
escape-attempts-weigh_10_5kg
name-ends-in_mon
empts](https://user-images.githubusercontent.com/39535023/212660551-56a935f3-5b0d-4f61-b3f1-c72e5948f393.png)
tps://user-images.gith
not-Gabumon
ubusercontent.com/39535023/212660532-2a3fbba3-966d-439c-95a5-aa4afa1933e4.png)

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Agumon', 'February 3, 2020', 0, TRUE, 10.23);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Gabumon', 'November 15, 2018', 2, TRUE, 8);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Devimon', 'May 12, 2017', 5, TRUE, 11);
```
Comment (Ghiftee) [inline data.sql:6]: - [ ] You are currently missing the data below. Please add it as it is a part of the requirements
Screenshot 2023-01-16 at 12 05 26 (2)

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_part('year', date_of_birth) BETWEEN 2016 AND 2019;
+SELECT name FROM animals WHERE neutered IS TRUE AND escape_attempts < 3;
+SELECT date_of_birth FROM animals WHERE name IN ('Augmon', 'Pikachu');
```
Comment (Ghiftee) [inline queries.sql:6]: - [ ] This query result is not correct as you do not have ```Pikachu``` data

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hello @kennankole  👋🏽 

Good job working on the project so far! 
To highlight 👍🏽 
✔️ You used the template file
✔️ You added screenshots of your queries
✔️ Your queries are successful

However, there are some issues that you need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: queries.sql (hunk 2 above)
Comment (kennankole) [reply queries.sql:6]: Hello @Ghiftee 
I think I only forgot to include the SQL command of `Pikachu` in `data.sql` but it is there in the table, so the query works just fine, below is  a screenshot to show this. 
Screenshot from 2023-01-16 14-29-00

#### Thank you

**Review by mckent05 — APPROVED** [review]
Hi @kennankole, 

# STATUS: APPROVED ✔️ ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and well done! 🎉 
approved


## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👏 👏 
- All query results display correctly 👍 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**